### ISSUE TITLE
Add scm file for Lua to enable repo map support

### ISSUE THREAD

**Issue by hhn12138** [Aider-AI/aider#833]
### Issue

The document states that aider's tree-sitter supports the Lua language, but I cannot find the scm file for extracting Lua function blocks in the project's query directory. Could you please tell me where I can find the scm file for extracting Lua function blocks?

**Comment by paul-gauthier** [comment]
Thanks for trying aider and filing this issue.

Sorry, I'm not sure where to find the lua query scm file.

Is there something that isn't working well for you when using aider with lua? It should work fairly well even without the scm files.

**Comment by hhn12138** [comment]
@paul-gauthier   Certainly! May I inquire as to why Tree-sitter does not necessitate an SCM file for the extraction of the repository map from the Lua code repository, and yet Aider is able to work without it?

**Comment by paul-gauthier** [comment]
Aider works fine with languages that don't have repo-map support. It may not work *as well*, but my suggestion is just to try and use aider and see rather than spending a bunch of effort up front worrying about repo-map support.

**Comment by paul-gauthier** [comment]
Please see these two related issues for relevant updates:
- https://github.com/Aider-AI/aider/issues/3133
- https://github.com/Aider-AI/grep-ast/issues/7
